core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/kubernetes/KubernetesClient.scala

```diff
@@ -139,9 +139,11 @@ class KubernetesClient(
           image: String,
           memory: ByteSize = 256.MB,
           environment: Map[String, String] = Map.empty,
-          labels: Map[String, String] = Map.empty)(implicit transid: TransactionId): Future[KubernetesContainer] = {
+          labels: Map[String, String] = Map.empty,
+          resourceTags: Option[List[String]],
+          imagePullSecret: Option[String] = None)(implicit transid: TransactionId): Future[KubernetesContainer] = {
```

Comment:
* resourceTags (support the runtime pod is created on specified nodes)
resourceTags  is invoked `WhiskPodBuilder.scala`'s `def buildPodSpec` , e.g 
```
    resourceTags.map { tags =>
      val affinity = specBuilder
        .editOrNewAffinity()
        .editOrNewNodeAffinity()
        .editOrNewRequiredDuringSchedulingIgnoredDuringExecution()
      affinity
        .addNewNodeSelectorTerm()
        .addNewMatchExpression()
        .withKey(KubernetesClient.OPENWHISK_RUNTIME_RESOURCE_ROLE)
        .withOperator("In")
        .withValues(tags.asJava)
        .endMatchExpression()
        .endNodeSelectorTerm()
        .endRequiredDuringSchedulingIgnoredDuringExecution()
        .endNodeAffinity()
        .endAffinity()
    }
```
* imagePullSecret
Just reserve a field, but not implement it, it is for blackbox action's image, when pull it, may need the secret.